### PR TITLE
Closes #5 lookup vote feature

### LINKED ISSUE #5: Receive Vote feature
full details at https://github.com/AgileVentures/AsyncVoter/blob/master/features/receive_vote.feature

### PR DESCRIPTION
**Pull request by sysrex**
Added the controller, and the test for the controller.

### CODE REVIEW

**Comment by joaopapereira** [comment]
Fixes #5

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+MONGODB=mongodb://localhost/asyncvoter
```
Comment (joaopapereira) [inline .env]: Ii believe this file should be created by each person locally right?

File: .idea/AsyncVoter.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joaopapereira) [inline .idea/AsyncVoter.iml]: This file should be removed

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joaopapereira) [inline .idea/misc.xml]: This file should be removed

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joaopapereira) [inline .idea/modules.xml]: This file should be removed

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joaopapereira) [inline .idea/vcs.xml]: This file should be removed

File: .idea/workspace.xml [hunk 6]
```diff
@@ -0,0 +1,412 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joaopapereira) [inline .idea/workspace.xml]: This file should be removed

File: src/story/story.routes.js [hunk 7]
```diff
@@ -13,6 +13,7 @@ router.use(function timeLog(req, res, next) {
 // Define the home page route
 router.get('/stories', controller.allStories);
 router.post('/stories', controller.createStory);
+router.get('/stories/:id', controller.findById);
```
Comment (joaopapereira) [inline src/story/story.routes.js]: I believe this is not part of the current PR

File: test/story/story.routes.spec.js [hunk 8]
```diff
@@ -6,66 +6,87 @@ let should = chai.should();
...
             Story.create({
-                url: "http://google.com",
-                size: "1",
-                name: "dummy entry"
-            },
-            function(err, story) {
-                if (err) throw err;
-                done();
-            });
+                    url: "http://google.com",
```
Comment (joaopapereira) [inline test/story/story.routes.spec.js]: The indentation doesn't look correct

File: test/story/story.routes.spec.js [hunk 9]
```diff
@@ -6,66 +6,87 @@ let should = chai.should();
...
                 res.body.length.should.be.eql(1);
                 done()
             });
     });
 
-
-})
\ No newline at end of file
+    it('GET /stories/:id', function (done) {
+        var newStory = new Story({
```
Comment (joaopapereira) [inline test/story/story.routes.spec.js]: I believe this is not part of the current PR

File: .env (hunk 1 above)
Comment (sysrex) [inline .env]: I changed the behaviour to edit locally because the file needs to exist in order for travois ci to be able to execute the tests

Comment (tansaku) [inline .env]: added these to travis setttings

Comment (tansaku) [inline .env]: .env can now be deleted

File: test/story/story.routes.spec.js (hunk 9 above)
Comment (tansaku) [inline test/story/story.routes.spec.js]: I think it is ...

**Comment by tansaku** [comment]
replace with #29

**Pull request closed without merging**